checkmackerelmetric/checkmackerelmetric.go

```diff
@@ -38,11 +45,14 @@ func parseArgs(args []string) (*mackerelMetricOpts, error) {
 	p, _ := arg.NewParser(arg.Config{}, &mo)
 	err := p.Parse(args)
 
-	if err == arg.ErrHelp {
+	switch {
+	case err == arg.ErrHelp:
 		p.WriteHelp(os.Stdout)
 		os.Exit(0)
-	}
-	if err != nil {
+	case err == arg.ErrVersion:
+		fmt.Println(mo.Version())
```

Comment:
It seems unnecessary to deliberately define it as a method since it references variables not present in the mackerelMetricOpts structure.

```suggestion
		fmt.Printf("version %s (rev %s)\n", version, revision")
```

Reply:
Yes I know... but unfortunatelly go-arg library needs Version() interface implementation to show it in help message.

Reply:
I've understood the contents mentioned in the PR comments. It seems fine as it is!